modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/Constants.java

```diff
@@ -56,4 +56,7 @@
     public static final String XML = ".xml";
     public static final String CARBON_HOME = "carbon.home";
     public static final String CARBON_HOME_VALUE = "../../";
+
+    //application json content type
+    public static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
```

Comment:
Constant with same value is already existing above. So remove this new constant and use the other one.

Reply:
Fixed with https://github.com/wso2/product-apim/pull/9882/commits/f0d515aa2ae21a722d14995f6c227f16ebdbc361